doc/licenses/apache_jna.txt

```diff
@@ -88,7 +88,7 @@ Apache 2 license:
       or a Contribution incorporated within the Work constitutes direct
       or contributory patent infringement, then any patent licenses
       granted to You under this License for that Work shall terminate
-      as of the date such litigation is filed.
+      as of the date, such litigation is filed.
```

Comment:
this is somebody else's license file, so it's important it to leave it verbatim and not make any edits in our copy

Reply:
Sure. Reverted the changes from this file.